payments-core/src/main/java/com/stripe/android/model/parsers/ElementsSessionJsonParser.kt

```diff
@@ -46,6 +47,13 @@ internal class ElementsSessionJsonParser(
         val linkSettings = json.optJSONObject(FIELD_LINK_SETTINGS)
         val linkFundingSources = linkSettings?.optJSONArray(FIELD_LINK_FUNDING_SOURCES)
 
+        val flags = json.optJSONObject(FIELD_FLAGS)?.let { flags ->
+            parseFlags(
+                json = flags,
+                relevantFlags = ElementsSessionFlags.entries.toList().map { it.flagValue }
+            )
+        } ?: emptyMap()
```

Comment:
we were not mapping elements session flags before. Reused the existing `parseFlags` method and allowed to provide a list of relevant flags to avoid holding hundreds of flags on the session object unnecessarily.